services-turf/src/main/java/com/mapbox/turf/TurfMeasurement.java

```diff
@@ -395,4 +396,47 @@ public static Point along(@NonNull LineString line, @FloatRange(from = 0) double
     }
     return bbox;
   }
+
+  /**
+   * Takes a {@link MultiPolygon} and measures each polygons perimeter in the specified units. if
+   * one of the polygons contains holes, the perimeter will also be included.
+   *
+   * @param multiPolygon geometry to measure
+   * @param units        one of the units found inside {@link TurfConstants.TurfUnitCriteria}
+   * @return total perimeter of the input polygons combined, in the units specified
+   * @see <a href="http://turfjs.org/docs/#linedistance">Turf Line Distance documentation</a>
+   * @since 1.2.0
+   */
+
+  /**
+   * Takes a {@link MultiPolygon} and measures each polygons perimeter in the specified units. if
+   * one of the polygons contains holes, the perimeter will also be included.
+   *
+   * @param multiPolygon geometry to measure
+   * @param units        one of the units found inside {@link TurfConstants.TurfUnitCriteria}
+   * @return total perimeter of the input polygons combined, in the units specified
+   * @see <a href="http://turfjs.org/docs/#linedistance">Turf Line Distance documentation</a>
+   * @since 1.2.0
+   */
+
+  /**
+   * Takes a {@link BoundingBox} and uses its coordinates to create a {@link Polygon}
+   * geometry.
+   *
+   * @param boundingBox a {@link BoundingBox} object to calculate with
+   * @return a {@link Polygon} object
+   * @see <a href="http://turfjs.org/docs/#bboxPolygon">Turf BoundingBox Polygon documentation</a>
+   * @since 4.7.0
+   */
+  public static Polygon bboxPolygon(BoundingBox boundingBox) {
```

Comment:
Here are a couple of other observations sorry that I did not notice during the last review:

[ArrayList() Constructs an empty list with an initial capacity of ten](https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#ArrayList--
).
So when we are doing a library implementation it is best to take these things into account.  I think using Arrays.asList() could be one of the solutions:
    return Polygon.fromLngLats(
                   Arrays.asList(
                     Arrays.asList( Point.fromLngLat(boundingBox.south(), boundingBox.west()),
                                             Point.fromLngLat(boundingBox.south(), boundingBox.east()),
                                             Point.fromLngLat(boundingBox.north(), boundingBox.east()),
                                            Point.fromLngLat(boundingBox.north(), boundingBox.west()),
                                            Point.fromLngLat(boundingBox.south(), boundingBox.west()))));

Now ... as I typed this. I noticed that latitudes are passed into Points before longitudes but it should be the other way around.  Strange that the tests did not catch that..

Reply:
Just pushed and re-based.

This is working for me:
```java
public static Polygon bboxPolygon(BoundingBox boundingBox) {
    return Polygon.fromLngLats(
        Arrays.asList(
            Arrays.asList(
                Point.fromLngLat(boundingBox.south(), boundingBox.west()),
                Point.fromLngLat(boundingBox.south(), boundingBox.east()),
                Point.fromLngLat(boundingBox.north(), boundingBox.east()),
                Point.fromLngLat(boundingBox.north(), boundingBox.west()),
                Point.fromLngLat(boundingBox.south(), boundingBox.west()))));
  }
```

Reply:
I see what you mean now @osana . I switched the coordinates.

```java
public static Polygon bboxPolygon(BoundingBox boundingBox) {
    return Polygon.fromLngLats(
      Arrays.asList(
        Arrays.asList(
          Point.fromLngLat(boundingBox.west(), boundingBox.south()),
          Point.fromLngLat(boundingBox.east(), boundingBox.south()),
          Point.fromLngLat(boundingBox.east(), boundingBox.north()),
          Point.fromLngLat(boundingBox.west(), boundingBox.north()),
          Point.fromLngLat(boundingBox.west(), boundingBox.south()))));
  }
```

I also adjusted the way that I build a `BoundingBox` object in my tests. Tests are passing.

Pushed and rebased.